### PR TITLE
index: document API additions, authorization

### LINKED ISSUE #183: Update index server spec with latest changes
See qabel/qabel-core#600

### PR DESCRIPTION
**Pull request by enkore**
Fixes #183

### CODE REVIEW

File: _docs/Qabel-Index.md [hunk 1]
```diff
@@ -39,21 +39,43 @@ email
...
 
 phone
-: Phone number (must be capable of receiving SMS)
+: Phone number (must be capable of receiving SMS). Incoming phone
+numbers are normalized to ITU-T E.164 according to the specified
+locale (via HTTP Accept-Language), if they are not already. It is
+recommended to normalize phone numbers on the client.
+
+: The server might reject phone numbers due to blacklisting of their
+country code.
```
Comment (julianseeger) [inline _docs/Qabel-Index.md]: does this result in a `400`?

File: _docs/Qabel-Index.md [hunk 2]
```diff
@@ -39,21 +39,43 @@ email
...
+
 ### Search
 
 * Resource: /api/v0/search/
-* Method: GET
-* Request data: field-value pairs (`field=value[&field=value...]`)
+* Methods: GET, POST
+* Request data (GET): field-value pairs (`field=value[&field=value...]`)
+* Request data (POST): JSON `{"query": [{"field": STR, "value": STR}, ...]}`
 * Response data: `{"identities": [identity, ...]}`
```
Comment (julianseeger) [inline _docs/Qabel-Index.md]: `{"identities": [identity, ...], "matches": [{"field": STR, "value": STR}, ...]}`

**Comment by roeslpa** [comment]
Are there any restrictions that prevent a brute force search for all phone numbers? And do we want to prevent this?

**Comment by enkore** [comment]
*updated

Fixes #183 #185

File: _docs/Qabel-Index.md (hunk 1 above)
Comment (enkore) [inline _docs/Qabel-Index.md]: The ":" *should* be correct (no way to be sure with markdown tbh) and place this paragraph on the same definition-list level as the preceding paragraph.

**Comment by audax** [comment]
See https://github.com/Qabel/qabel-index/issues/48 for @roeslpa's issue

**Pull request merged**